### DISCUSSION TITLE
Add shorthand for simply adding theme variant

### DISCUSSION THREAD

**Discussion by silveltman** [RyanClementsHax/tailwindcss-themer#69] [Ideas]
I just added the following themes to my project:

```
require('tailwindcss-themer')({
	defaultTheme: {
		extend: { ... }
	},
	themes: [
                 {
                         name: 'my-theme',
                         extend: { ... }
                 },
		{
			name: 'size-sm',
		},
		{
			name: 'size-md',
		},
		{
			name: 'size-lg'
		},
	]
}),
```

What this allows me to is style my with classes like `size-sm:text-sm`, size-sm:p-4` and enable this by simply setting class `size-sm` on the element or a parent element.

I personally use this to create different sizes buttons and headings for example. Pretty cool imo!

Which brings me to the feature request, create a shorthand for this config like so:

```
require('tailwindcss-themer')({
	defaultTheme: {
		extend: { ... }
	},
	themes: [
                 {
                         name: 'my-theme',
                         extend: { ... }
                 },
                 'size-sm',
                 'size-md',
                 'size-lg',
	]
}),
```

**Comment by RyanClementsHax** [comment]
Oh I see! You're using themes to conditionally enable/disable certain inline styles....clever.

Thanks for pointing this out.

I think this is a reasonable request. Just to be sure I understand, you're looking for a shorthand to specify themes without any styles solely for the use of enabling/disabling inline styles?
